### PR TITLE
Predict phishing incidents using the out-of-the-box pretrained model

### PR DESCRIPTION
**Pull request by lperryp**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22138

## Description
Predict phishing incidents using the out-of-the-box pretrained model.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                                  'modelName': OUT_OF_THE_BOX_MODEL_NAME})
+    if is_error(res):
+        return_error(get_error(res))
+
+
+def predict_phishing_words(email_subject, email_body, email_body_html, min_text_length, label_threshold,
+                           word_threshold, top_word_limit, is_return_error):
+    if not oob_model_exists() or True:
+        load_oob_model()
+    res = demisto.executeCommand('DBotPredictPhishingWords', {'modelName': OUT_OF_THE_BOX_MODEL_NAME,
```
Comment (erezh31) [inline Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.py]: you can use `args = demisto.args()` and just change the modelName argument.

File: Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
+args:
```
Comment (erezh31) [inline Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.yml]: make sure you copy the exact argument from DBotPredictPhishingWords

File: Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        encryped_model = input_file.read()
+    encoded_model = demisto_ml.decrypt_model(encryped_model)
+    res = demisto.executeCommand('createMLModel', {'modelData': encoded_model,
+                                                   'modelName': OUT_OF_THE_BOX_MODEL_NAME,
+                                                   'modelLabels': ['legit', 'spam', 'malicious'],
+                                                   'modelOverride': 'true'})
+    if is_error(res):
+        return_error(get_error(res))
+
+    with open(EVALUATION_PATH, 'r') as json_file:
```
Comment (erezh31) [inline Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.py]: I think it's better to change this logic, and do everything in the docker image - create function `get_oob_model()` that will load and decrypt and return the model

**Comment by lperryp** [comment]
@roysagi - only the changelog is new, the rest is from an already existing script. Can you review it? thanks

**Pull request merged**